resources/fixtures/config/barcode.yml

```diff
@@ -1,3 +1,9 @@
 file-type: 'png'
 barcode-type: 'code39'
+
+# determines the class that will be used to automatically assign barcodes to products, will typically be the same as
+# the barcode-type
+barcode-generator: 'code39'
```

Comment:
Yeah, so you might want to use a different generator for that barcode type. Like EAN13 is actually more complex than Mothership makes out, or it might be a case that you want to connect to an API and get the official barcode for that product, but it would still be an EAN13 barcode